Makefile

```diff
@@ -149,6 +151,12 @@ endif
 	touch $@
 
 lint: golangci-lint
+ifeq ($(PRE_COMMIT),)
+	@echo "FATAL: pre-commit was not found, please install it." >&2
+	@exit 2
```

Comment:
This isn't helpful: many people will have no idea what pre-commit is or how to install it.

Please put yourself in the shoes of your target audience: take a step back, try to imagine what this will be like for others. Offer an exact command that they can copy/paste, such as a `make` target or a `pip` command.

Reply:
I was hoping for clearer instructions, but this will do.